### DISCUSSION TITLE
Request for a '@' icon

### DISCUSSION THREAD

**Discussion by nicedisguise** [twilio-labs/paste#1450] [Request: Icon] [answered]
We will likely need an icon for the '@' symbol as where we want to use it does not support text. 

<img width="1188" alt="@ icon request" src="https://user-images.githubusercontent.com/74921546/118268771-b72a1200-b4b5-11eb-973c-29198ca817fa.png">

**Comment by SiTaggart** [comment, marked as answer]
👋🏼 Hi @nicedisguise 

Are you referring to the input prefix and suffix?

Both the prefix and suffix on all input components, textarea, select and combobox will take any content. A text based `@` will be perfectly fine if that is the case.

https://paste.twilio.design/components/input#input-with-add-ons-prefix-suffix-text-or-icons

**Reply by kate-porter** [reply]
Hey @nicedisguise! Just wanted to check in on this to make sure Simon's reply answered your question.

Also, if it's helpful, here's a quick vid on how to switch between a text vs. icon prefix/suffix on form components in Figma: https://www.loom.com/share/ddc5ba034fc64763864f18507c669749

**Reply by nicedisguise** [reply]
hey, my bad sorry I neglected to respond. Thanks for the clarification both - we will use the font family rather than an icon

**Reply by kate-porter** [reply]
Awesome, glad that'll work for ya!
